alws/crud/errata.py

```diff
@@ -404,6 +401,10 @@ def errata_records_to_oval(records: List[models.NewErrataRecord]):
                 oval.append_object(
                     get_object_cls_by_tag(obj["type"]).from_dict(obj)
                 )
+    # currently only Almalinux8 have multiple GPG keys. So only this platfrom requires
+    # additional oval processing
```

Comment:
Please do keep a link to the [original report](https://github.com/AlmaLinux/build-system/issues/205), so anybody reading this code understands why we have to perform this additional processing

Reply:
@javihernandez added  link to report.
Thanks for pointing out)